### PR TITLE
Add an export presenter.

### PR DESCRIPTION
**Pull request by reiterl**
Needs a meeting_id and uses export_meeting helper to export the meeting
of the meeting_id.

### CODE REVIEW

File: openslides_backend/presenter/__init__.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from . import (  # noqa
     check_mediafile_id,
     check_resource_id,
+    export,
```
Comment (jsangmeister) [inline openslides_backend/presenter/__init__.py:4]: Maybe `export_meeting` is more concise?

**Review by jsangmeister — CHANGES_REQUESTED** [review]
Change name and add migration index to the exported file. You can find the method for this here: https://github.com/OpenSlides/openslides-backend/blob/f0fe824b8199011543732114240da11334828f3e/migrations/__init__.py#L32 You have to split this method up and extract the datastore migration index.

File: openslides_backend/presenter/__init__.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/presenter/__init__.py]: Okay, I renamed it into `export_meeting`.

**Comment by reiterl** [comment]
Add _migration_index to export meeting presenter. Extract the way to get the mig index from the assert_migration_index()

**Review by jsangmeister — CHANGES_REQUESTED** [review]
Did not realize that both migration indices must be the same so you can of course also use the backend migration index. But while checking this, I noticed that an `assert_migration_index` is still missing for the presenter view at this line: https://github.com/OpenSlides/openslides-backend/blob/811cde32605ab62799b0c37f557a8540dd6ed03c/openslides_backend/http/views/presenter_view.py#L20 (analogous to https://github.com/OpenSlides/openslides-backend/blob/2c1493c2c10143d9ed0d9a6a5fc362ff5169eced/openslides_backend/http/views/action_view.py#L26)

Please add this to make sure that the migration indices are alway equal.

**Review by jsangmeister — APPROVED** [review]

**Pull request merged**